cmake/OpenCVDownload.cmake

```diff
@@ -98,6 +139,8 @@ function(ocv_download)
     endif()
   endforeach()
 
+  ocv_cmake_hook(OCV_DOWNLOAD_MIRROR_${OPENCV_DOWNLOAD_HOST})
```

Comment:
Hook could update DL_HASH value, so its call should be moved upper (at least before "# Check CMake cache for already processed tasks").

Reply:
Done.